### DISCUSSION TITLE
Variables arent replaced

### DISCUSSION THREAD

**Discussion by SamTV12345** [wiverson/jtoolprovider-plugin#6] [General]
I have the problem that the environment variables arent replaced. The target/packaging/win-jpackage.txt looks like this:

```
--type msi
--name ${app.name}
--win-menu
--win-menu-group ${windows.vendor}
--vendor ${windows.vendor}
--icon ${project.basedir}\app-icon.ico
--dest ${project.build.directory}
--main-jar ${project.build.finalName}.jar
--main-class ${main-class}
--input ${project.build.directory}\dependency
--app-version ${app.version}
--runtime-image ${project.build.directory}\jvm-image
--temp ${project.build.directory}\installer-work
--win-upgrade-uuid ${windows.upgrade.uuid}
--description "${project.description}"
--copyright "(C) ${windows.vendor}"
```

and I'm getting this error:

[ERROR] jpackage failed with error code [1]
[INFO] Specified runtime image directory runtime-image: ${project.build.directory}\jvm-image does not exist

**Comment by wiverson** [comment]
Go ahead and post the rest of the build - something may be failing earlier.

**Reply by SamTV12345** [reply]
Thank you for helping me. This is my repository: https://bitbucket.org/Samuel19982/ueb2client/src/master/

You should be able to use mvn install rightaway.

**Reply by wiverson** [reply]
Huh. Took a quick look, looks like you changed a lot from the original, including adding Gradle and also not pointing at the target directory anymore for the output, but the rest of the project is still looking for the target directory. FWIW I noticed that the POM declares a minimum version of 2.2.0, I would use the current version of Maven.

Also, I'll note that merging Spring Boot + JavaFX is not a starter project kind of thing - they have pretty different models and can be very difficult to get working together. Not impossible, but surprisingly not easy either.

**Comment by SamTV12345** [comment]
Thank you for taking a look at my source code. I removed the prerequisites tag. I guess my project is too complex for this plugin.
